securedrop/sdconfig.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+import typing
+# https://www.python.org/dev/peps/pep-0484/#runtime-or-type-checking
+if typing.TYPE_CHECKING:
+    # flake8 can not understand type annotation yet.
+    # That is why all type annotation relative import
+    # statements has to be marked as noqa.
+    # http://flake8.pycqa.org/en/latest/user/error-codes.html?highlight=f401
+    from typing import List, Dict  # noqa: F401
+
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        try:
+            self.FlaskConfig = _config.FlaskConfig  # type: ignore
```

Comment:
please do not include modules, only variables

Reply:
Why not? These modules impact how the app behaves and we need to be able to inject them in tests.

Reply:
Could you please show me where in the code it would make a difference? I honestly thought it was entirely unecessary.

Reply:
@dachary That is not a module, it is class which is part of current `config.py`.

Reply:
correct, but it is only used internally and not anywhere else in the code so let's not needlessly expose it

Reply:
I think we should keep everything exposed exactly as it was in this PR and then a second one to restructure it.

Reply:
Wait, sorry. You're right. This one is just a subclass of the things we use. I misunderstood you. Yes, this one should not be part of `SdConfig`.

Reply:
@heartsucker @dachary Okay, will remove that one.